docs/SUMMARY.md

```diff
@@ -52,6 +52,7 @@
       - [Purge History](admin_api/purge_history_api.md)
       - [Purge Rooms](admin_api/purge_room.md)
       - [Register Users](admin_api/register_api.md)
+      - [Registration Tokens](admin_api/registration_tokens.md)
```

Comment:
I think the location of new docs should be there.

```suggestion
      - [Registration Tokens](usage/administration/admin_api/registration_tokens.md)
```

See there: https://github.com/matrix-org/synapse/blob/develop/docs/admin_api/README.rst

Reply:
Thanks